doc/src/modules/discrete.rst

```diff
@@ -93,3 +93,14 @@ Intersecting Product
 ^^^^^^^^^^^^^^^^^^^^
 
 .. autofunction:: intersecting_product
+
+
+Recurrences
+-----------
+
+.. module:: sympy.discrete.recurrences
+
+Linear Recurrences
+^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: linrec
```

Comment:
This could be excluded for now, it should have a user-facing method before making it public.

Reply:
Removed it, have a look!